src/main/java/teammates/ui/template/FeedbackResultsTable.java

```diff
@@ -35,9 +51,37 @@ public FeedbackResultsTable(int fbIndex, String studentName, FeedbackSessionResu
             for (Map.Entry<String, List<FeedbackResponseAttributes>> entry : given.entrySet()) {
                 recipientIndex++;
                 this.givenResponses.add(new FeedbackResponsePersonRow(fbIndex, recipientIndex, entry.getKey(), "recipient",
-                                                                      entry.getValue(), result));
+                                                                      entry.getValue(), result, false));
+            }
+        }
+    }
+
+    private FeedbackResponseAttributes findContriFeedbackResponse(
+            Map<String, List<FeedbackResponseAttributes>> rcvedFeedbacks) {
+        FeedbackResponseAttributes[] newResponseDetails = new FeedbackResponseAttributes[1];
+        boolean[] hasContriFeedbackResponseRow = {false};
+
+        rcvedFeedbacks.entrySet().forEach(entry -> {
+            for (FeedbackResponseAttributes questionResponses : entry.getValue()) {
```

Comment:
You may want a nested lambda to reduce the nesting depth here:
```java
rcvedFeedbacks.forEach((key, entry) -> entry.getValue().forEach(questionResponses -> {
    ...
}));
```

Reply:
That definitely makes it clearer. Thanks.